TUT-103 : Added login screen

preetipdapkekar opened the pull request:
- Modified activity_main.xml login screen
  Added linear layouts for user name and password
  Added login button
- Added dimen.xml file for dimentions
  Added dimentions for textfield
  Added margin for layout

preetipdapkekar committed:
TUT-103 : Added login screen

- Modified activity_main.xml login screen
  Added linear layouts for user name and password
  Added login button
- Added dimen.xml file for dimentions
  Added dimentions for textfield
  Added margin for layout

preetipdapkekar commented:
![login_screen](https://user-images.githubusercontent.com/43718349/46257347-a6c7f300-c4d5-11e8-8e8c-1e1682893acd.png)

theyogeshrathod commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -1,18 +1,69 @@
+
+    </LinearLayout>
+
+
```
Remove this extra line.

preetipdapkekar replied:
okay ..I'll do.

theyogeshrathod commented on app/src/main/res/values/dimen.xml:
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="login_layout_margin">20dp</dimen>
+    <dimen name="user_name">20sp</dimen>
```
Rename `user_name -> user_name_text_size`

theyogeshrathod commented on app/src/main/res/values/dimen.xml:
```diff
@@ -0,0 +1,6 @@
+<resources>
+    <dimen name="login_layout_margin">20dp</dimen>
+    <dimen name="user_name">20sp</dimen>
+    <dimen name="password">20sp</dimen>
```
Rename `password  -> password _text_size`

preetipdapkekar replied:
ok. I will do all the changes.

theyogeshrathod commented:
Please make the changes in the same branch:
Follow these steps:
- git checkout pr/preetipdapkekar/TUT-103-create-login-screen-byPreeti
- git pull
- Make changes in Android Studio
- git diff
- git add .
- git commit
- git push origin HEAD

Thank you.

archana-kalesan committed:
- Made changes in activity_main.xml
   Removed extra line
 - Made changes in dimen.xml
   password dimen name renmaed as password_text_size
   User name dimen renamed as username_text_size

preetipdapkekar commented:
okay

preetipdapkekar committed:
Removed gravity of EditText

theyogeshrathod commented:
Thank you for the change. I'm merging this pull-request.
